File: MT.Model/Dictionary/AspNetUsers.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region AspNetUser:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a AspNetUser:EntityBase.
    /// </summary>
    public class AspNetUsers : EntityBase
    {
        [Key]
        public int Id { get; set; }
        public Int32 AspNetUsersID { get; set; }
        public String AspNetRolesName { get; set; }
        public String CompanyParentName { get; set; }
        public String CompanyName { get; set; }
        public string Email { get; set; }
        public bool EmailConfirmed { get; set; }
        public string PasswordHash { get; set; }
        public string SecurityStamp { get; set; }
        public string PhoneNumber { get; set; }
        public bool PhoneNumberConfirmed { get; set; }
        public bool TwoFactorEnabled { get; set; }
        public DateTime LockoutEndDateUtc { get; set; }
        public bool LockoutEnabled { get; set; }
        public int AccessFailedCount { get; set; }
        public string UserName { get; set; }
        public int GroupID { get; set; }
        public int UserID { get; set; }
        public string FullName { get; set; }
        public int oId { get; set; }
        public string Address { get; set; }

    }
    #endregion
}

File: BECore/app/model/AspNetUsers.js
/// <summary>
/// Model EXTJS AspNetUsers.
/// </summary>
Ext.define('QLDT.model.AspNetUsers', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    idType: 'Guid',
    fields: [
        { name: "Id", type: "int" },
        { name: "Email", type: "string" },
        { name: "EmailConfirmed", type: "boolean" },
        { name: "PasswordHash", type: "string" },
        { name: "SecurityStamp", type: "string" },
        { name: "PhoneNumber", type: "string" },
        { name: "PhoneNumberConfirmed", type: "boolean" },
        { name: "TwoFactorEnabled", type: "boolean" },
        { name: "LockoutEndDateUtc", type: "date" },
        { name: "LockoutEnabled", type: "boolean" },
        { name: "AccessFailedCount", type: "int" },
        { name: "UserName", type: "string" },
        { name: "GroupID", type: "int" },
        { name: "UserID", type: "int" },
        { name: "FullName", type: "string" },
        { name: "Address", type: "string" },
        { name: "ModifiedDate", type: "date" },
        { name: "CreatedDate", type: "date" },
        { name: "IPAddress", type: "string" },
        { name: "ModifiedBy", type: "string" },
        { name: "CreatedBy", type: "string" },
        { name: "oId", type: "int" },
    ]
});

